IFC4x3/Templates/Product Shape/Product Geometric Representation/Body Geometry/Body AdvancedSweptSolid Geometry/Body SectionedSolidHorizontal/DocTemplateDefinition.xml

```diff
@@ -16,18 +16,7 @@
 										</DocModelRuleAttribute>
 										<DocModelRuleAttribute CardinalityMin="1" CardinalityMax="1" Name="RepresentationIdentifier" Identification="Identifier">
 											<Rules>
-												<DocModelRuleEntity id="IfcLabel_2nTsBx4yb0wfI4IH_ymbNg" Name="IfcLabel" UniqueId="b17762fb-13c9-40ea-9484-491fbcc255ea">
-													<Rules>
-														<DocModelRuleConstraint>
-															<Expression xsi:type="DocOpStatement" Operation="compareequal">
-																<Reference xsi:type="DocOpReference">
-																	<EntityRule xsi:type="DocModelRuleEntity" xsi:nil="true" href="IfcLabel_2nTsBx4yb0wfI4IH_ymbNg" />
-																</Reference>
-																<Value xsi:type="DocOpLiteral" Operation="loadstring" Literal="Body" />
-															</Expression>
-														</DocModelRuleConstraint>
-													</Rules>
-												</DocModelRuleEntity>
+												<DocModelRuleEntity Name="IfcLabel" UniqueId="b17762fb-13c9-40ea-9484-491fbcc255ea" />
```

Comment:
As far as I can tell @larswik the rules should be present in the serialisation of the sub templates, this is the case with the mvdXML encoding also subtemplates redefine the tree and rules of the parent and must match to be valid.

Hence why the rule is serialised on this template also. so please revert 134270f as the previous was the correct serialisation